recipe/meta.yaml

```diff
@@ -68,3 +68,5 @@ extra:
     - tomaugspurger
     - jrbourbeau
     - charlesbluca 
+    - fjetter
+
```

Comment:
```suggestion
    - fjetter
```

Reply:
Hrm, that didn't quite work.  Anyway, @fjetter, the linter is complaining because of the extra line break here.

Reply:
Happy to merge afterwards